src/OpenQasmReader/Parser.cs

```diff
@@ -671,7 +739,7 @@ private static void WriteOperation(StringBuilder outside, Dictionary<string, int
                 foreach (var qubitRegister in qRegs)
                 {
                     Indent(outside);
-                    outside.AppendFormat("using ({0} = Qubit[{1}]) {{\n", qubitRegister.Key, qubitRegister.Value);
+                    outside.AppendFormat("use {0} = Qubit[{1}] {{\n", qubitRegister.Key, qubitRegister.Value);
```

Comment:
Is incorrect. Will give "use q = Qubit[3] { " instead of the correct "use q = Qubit[3];".